CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## vNext
 
 - Removed auto-payout and participation in governance filters
+- Added signed extrinsic cumulative and bar charts
```

Comment:
Let's split into 2:
1. Implemented charts framework for cumulative line and bar types
2. Implemented `Signed Extrinsics` charts.

Reply:
done